src/byte_set.rs

```diff
@@ -11,19 +11,22 @@ pub struct ByteSet {
 }
 
 impl ByteSet {
-    pub fn new() -> ByteSet {
+    #[inline(always)]
```

Comment:
Is this inline declaration necessary for use in const? Is the const keyword not enough?

Reply:
The const keyword is enough, and the compiler might inline even without the attribute. I can remove it if it is excessive.

Reply:
Removed, it can always be added back if appropriate.